docs/en/advanced_guides/customize_dataset.md

```diff
@@ -20,35 +20,35 @@ Currently, we only support '.bin' format point cloud for training and inference.
 
```

Comment:
Currently, we only support '.bin' format -> Currently, we only support `.bin` format